fsck/fsck.c

```diff
@@ -1206,8 +1207,6 @@ int main(int argc, char * const argv[])
 	struct exfat *exfat = NULL;
 	bool version_only = false;
 
-	print_level = EXFAT_ERROR;
```

Comment:
@hclee Hyunchul, should we replace exfat_info with exfat_debug instead of set print_level with EXFAT_ERROR here ?

Reply:
@namjaejeon Yes, but we need to print more useful information. I will implement it.
Hello @luca020400, this variable is already declared as static in exfat_tools.h

Reply:
The main issue is that the variable is included in every file that includes the header, this becoming unused and some compilers warn about it.

Reply:
Ah, Okay. Can you change like this ?

```
diff --git a/include/exfat_tools.h b/include/exfat_tools.h
index 45de4aa..57071f1 100644
--- a/include/exfat_tools.h
+++ b/include/exfat_tools.h
@@ -76,7 +76,7 @@ ssize_t exfat_write(int fd, void *buf, size_t size, off_t offset);
  * Exfat Print
  */
 
-static unsigned int print_level = 1;
+extern unsigned int print_level;
 
 #define EXFAT_ERROR    (0)
 #define EXFAT_INFO     (1)
diff --git a/lib/libexfat.c b/lib/libexfat.c
index 1177cf0..5f8d824 100644
--- a/lib/libexfat.c
+++ b/lib/libexfat.c
@@ -27,6 +27,8 @@
 #define BIT_MASK(nr)            ((1) << ((nr) % 32))
 #define BIT_WORD(nr)            ((nr) / 32)
 
+unsigned int print_level  = EXFAT_INFO;
```